spec/features/devise/sessions_spec.rb

```diff
@@ -0,0 +1,73 @@
+#Feature: User Session
+#  As a current user
+#  I want to sign in and out
+#  So I can access protected pages.
+feature "Sessions" do
+
+  #Scenario: Visit the log in page
+  #  Given I am a current user
+  #  When I attempt to log in with a blank password
+  #  Then I see an error message for invalid fields.
+  scenario "Users can't sign in with blank password" do
+    visit root_path
+    fill_in 'Email', with: 'mail@mail.com'
```

Comment:
FYI, I added a session_helper.rb into spec/helpers that simplifes some of these.